### ISSUE TITLE
WS Comply Attest Integration

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1431]
To automate accessibility testing and bring it into our existing testing framework, integrate WorldSpace Comply accessibility checker with heliotrope. Currently only Jon and Seth have access to their gem for initial integration into the application.

- [ ] add relevant gem to Gemfile
- [ ] create basic spec to test if working correctly
- [ ] decide where to store test result output (JSON and HTML files)
- [ ] to begin, ensure that failing accessibility specs are not breaking the build

As we remove accessibility failures, we'll want to change the spec so that builds will break when new accessibility issues are introduced to the application.

**Comment by jmcglone** [comment]
There is separate documentation on the agora Deque site that wants you to customize your `~/.gemrc` file with a source and login credentials, which to me implies this is a global gem only installed on one's local machine.  But since we manage gems at the project level, we probably need to use Bundler to grab the gem, right?

So, fetching gems from a private server using bundler:

```
bundle config https://agora.dequecloud.com/artifactory/api/gems/attest-ruby/ user:password
```
 "...the command must be run once on each computer that will install the Gemfile, but this keeps the credentials from being stored in plain text in version control..."

@sethaj so, how exactly do we do this so that all fulcrum devs can have the a11y test framework/not break everyone's local environment? 

In `Gemfile` do:
```
gem 'attest-ruby',  :source => 'https://agora.dequecloud.com/artifactory/api/gems/attest-ruby/'
```

**Comment by jmcglone** [comment]
In looking around agora more, there is a direct download of the gem, which could be an alternative way of setting this up for us.

**Comment by jmcglone** [comment]
We decided to not do this the Deque way. There is an out of date branch `axe-matchers` that integrates an open source gem that is the equivalent to Deque's Attest tool.

What is missing on that branch are the initial tests to run on pages.
